### ISSUE TITLE
Installing mxnet using pip works!

### ISSUE THREAD

**Issue by kayser-soze** [msracver/Flow-Guided-Feature-Aggregation#68]
TL;DR: you can install mxnet using pip instead of compile it from source!

For all the ones having trouble to complie mxnet from source:

I did not manage to compile mxnet from source and encountered several problems mentioned in other issues. Then I tried to compile a newer version of mxnet (official 1.5.1). Doing so I noticed that the operator files already exist in the official mxnet source. 
Thus, I installed this mxnet version using pip - I have Cuda 10.0 so I've used the following command line:

pip install mxnet-cu100==1.5.1

and it works!

After that, I had out of memory problems while running demo.py. I have GTX 1080 with 8GB of RAM. I dealt with it by changing KEY_FRAME_INTERVAL from 9 to 5 (in fgfa_rfcn/demo.py), and changing the code in demo.py as in https://github.com/msracver/Flow-Guided-Feature-Aggregation/issues/16#issuecomment-444388962

**Comment by wufengbin123** [comment]
Hello , I installed pip install mxnet-cu100==1.5.1, and  run demo.py successfully,but when I run train.py,after training. I met the error. 

Traceback (most recent call last):
  File "experiments/fgfa_rfcn/fgfa_rfcn_end2end_train_test.py", line 20, in <module>
    train_end2end.main()
  File "experiments/fgfa_rfcn\..\..\fgfa_rfcn\train_end2end.py", line 186, in main
    config.TRAIN.begin_epoch, config.TRAIN.end_epoch, config.TRAIN.lr, config.TRAIN.lr_step)
  File "experiments/fgfa_rfcn\..\..\fgfa_rfcn\train_end2end.py", line 179, in train_net
    arg_params=arg_params, aux_params=aux_params, begin_epoch=begin_epoch, num_epoch=end_epoch)
  File "experiments/fgfa_rfcn\..\..\fgfa_rfcn\core\module.py", line 999, in fit
    self.set_params(arg_params, aux_params)
  File "D:\Software\Anaconda3\envs\FGF\lib\site-packages\mxnet\module\base_module.py", line 699, in set_params
    allow_extra=allow_extra)
TypeError: init_params() got an unexpected keyword argument 'allow_extra'

can anyone tell me how to fix it? thanks very much!

**Comment by ZhijunHou** [comment]
Just set allow_extra=True.

**Comment by wufengbin123** [comment]
> Just set allow_extra=True.

Thanks for your help.but when I tried to change allow_extra=allow_extra with allow_extra=True in mxnet\module\base_module.py line 699,the problem is still there.  then I also change    
def set_params(self, arg_params, aux_params, allow_missing=False, force_init=True,
                   allow_extra=False):     
with
def set_params(self, arg_params, aux_params, allow_missing=False, force_init=True,
                   allow_extra=True):        
in mxnet\module\base_module.py line 671,
the problem is still there .

**Comment by joe660** [comment]
> > Just set allow_extra=True.
> 
> Thanks for your help.but when I tried to change allow_extra=allow_extra with allow_extra=True in mxnet\module\base_module.py line 699,the problem is still there. then I also change
> def set_params(self, arg_params, aux_params, allow_missing=False, force_init=True,
> allow_extra=False):
> with
> def set_params(self, arg_params, aux_params, allow_missing=False, force_init=True,
> allow_extra=True):
> in mxnet\module\base_module.py line 671,
> the problem is still there .
您这个问题解决了 我遇到了同样的问题  希望得到您的帮助，感谢。

**Comment by joe660** [comment]
> Just set allow_extra=True.

这个问题有什么好的解决方法吗？谢谢回复
